src/Queries/Where/Where.ts

```diff
@@ -232,15 +232,18 @@ export class Where {
             operator: 'eq',
           });
         } else if (value !== null && value !== undefined) {
-          for (const operator of operators) {
+          if (typeof value !== 'object') continue;
+          const symbols = Object.getOwnPropertySymbols(value);
+          for (const symbol of symbols) {
+            const operator = symbol.description;
+            if (!operator || !operators.includes(operator)) continue;
             if (isOperator[operator](value)) {
```

Comment:
Same here, merge those two?
```suggestion
            if (operator && operators.includes(operator) && isOperator[operator](value)) {
```